src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ *  (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ *  (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
-}
+  const result = {};
+  const resultArr = [];
+  const strArr = sourceString.split(';').join(':').split(':');
```

Comment:
```suggestion
  const strArr = sourceString.split(';').join(':').split(':');
```
зачем делать join(:).split(:)?